pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1717,15 +1717,6 @@ func ValidateVirtualMachineInstanceMetadata(field *k8sfield.Path, metadata *meta
 		}
 	}
 
-	if _, exists := annotations[v1.CPUManagerPolicyBetaOptionsAnnotation]; exists && !config.CPUManagerPolicyBetaOptionsEnabled() {
```

Comment:
Commit message:
> Currently the VMI is blocked if the annotation

Currently, the VMI creation is block if the `<annotationname>` annotation is set on the VMI..

> pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go
pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

Seem to be leftovers

Reply:
DONE